examples/problem_decomposition/mifno.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
     "# Exploring the Method of Increments with QEMIST Cloud and Tangelo\n",
     "\n",
-    "In this notebook, we illustrate how users can leverage QEMIST Cloud and Tangelo to explore the impact of quantum computing on problems tackled with the MI-FNO problem decomposition technique.\n",
+    "In this notebook, we illustrate how users can leverage QEMIST Cloud and Tangelo to explore the impact of quantum computing on problems tackled with the MI-FNO problem decomposition technique. \n",
```

Comment:
Maybe let's introduce this as the Method of Increment (MI) and explicitly mention 2 of its different flavours we are known for: MI-FNO and iFCI (you may use links to an external blog post or website article for which we are known, to establish the connection for the reader and highlight our achievements). 

Your section that describes MI-FNO with the drawing can then introduce the small differences between the two.